app/src/bf-game/index.js

```diff
@@ -187,16 +215,27 @@ export function playCard(gameState, card) {
     newGameState.turnCount++;
   } else if (card === CardType.Shuffle) {
     newGameState.deck = shuffle(gameState.deck);
-  }
+  } else if (card === CardType.Loot) {
+    // set looter
+    newGameState.looterId = userId;
+    const opponentsAlive = getOpponentsAlive(gameState);
 
-  const playerId = getCurrentPlayerId(gameState);
-  const playerHand = newGameState.hands[playerId];
-  const cardIndex = playerHand.indexOf(card);
-  playerHand.splice(cardIndex, 1)[0];
+    console.log(userId + " is looter");
+
+    newGameState.specialPhase = GamePhase.ChoosingLootTarget;
 
-  console.log("player " + playerId.substring(0, 8) + " played " + card);
+    if (opponentsAlive.length === 1) {
+      console.log("Only 1 opponent alive, auto select loot victim");
+      newGameState.specialPhase = GamePhase.ResolvingLoot;
+      newGameState.lootTargetId = opponentsAlive[0];
+    } else {
+      newGameState.specialPhase = GamePhase.ChoosingLootTarget;
+    }
+  }
 
-  newGameState.discardPile.unshift(card);
+  if (discard) newGameState.discardPile.unshift(card);
```

Comment:
You can check `gameState.specialPhase === GamePhase.ResolvingLoot` instead of using a new variable
```suggestion
  // dont send to discard pile, as card has been transfered to the looter
  if (gameState.specialPhase === GamePhase.ResolvingLoot) newGameState.discardPile.unshift(card);
```